### DISCUSSION TITLE
Show the seconds for modified timestamp in the file list.

### DISCUSSION THREAD

**Discussion by spinjector** [aonez/Keka#1217] [Ideas]
Hello, can we show the seconds for the file modified timestamp in the main file list? I do not see a way to see seconds without going to file properties. Thank you.

![CDViewer_ZYXNO8CNhw](https://user-images.githubusercontent.com/21039987/218195050-09937920-c7f8-4b98-9fa4-37edb92565c4.png)

**Comment by aonez** [comment]
Are you taking about Keka for iOS? Or maybe you’re talking about 7-Zip?
